Makefile

```diff
@@ -0,0 +1,20 @@
+# Makefile for building and testing documentation in a docker container
+#
+
+.PHONY: help
+help:
+	@echo "  docker-preview     to build live preview of docs using sphinx-autobuild in a docker container"
+	@echo "  docker-test        to build and test docs in a docker container"
+
+# Build live preview docs in a docker container
+.PHONY: docker-preview
+docker-preview:
+	make -C docs clean
+	DOCKER_RUN_ARGS="-p 0.0.0.0:8000:8000" ./docs/doc_scripts/docker-docs.sh \
```

Comment:
This builds the docs locally using our containthedocs docker image; 'preview' uses sphinx-autobuild to serve a local copy and rebuild as you write. So, if you want to work on a doc and be able to see your changes as you go, run `make docker-preview` and then open  0.0.0.0:8000 in your browser. :D